tests/libnet/ipaddress.go

```diff
@@ -0,0 +1,29 @@
+package libnet
+
+import (
+	v1 "k8s.io/api/core/v1"
+	netutils "k8s.io/utils/net"
+)
+
+func GetIp(ips []string, family v1.IPFamily) string {
+	for _, ip := range ips {
+		if netutils.IsIPv6String(ip) {
+			if family == v1.IPv6Protocol {
+				return ip
+			}
+		} else {
+			if family == v1.IPv4Protocol {
+				return ip
+			}
+		}
+	}
+	return ""
+}
+
+func GetPodIpsStrings(podIPs []v1.PodIP) []string {
```

Comment:
i would suggest to remove this function and just use ip.IP in the caller `GetIp` please

Reply:
But is is an array of `PodIp`s.

Reply:
why does it matter ? i mean there is no need for an helper
func GetPodIpByFamily(pod *v1.Pod, family v1.IPFamily) string {
	return getIp(convertPodIpsToStrings(pod.Status.PodIPs), family)
}

i dont see why getIp needs a helper convertPodIpsToStrings
it can do itself, unless you have (i didnt find?) one time that it get ip list and one time it get PodiP

Reply:
Will move it to `GetPodIpByFamily`